.kotlin_alpha/services/athena/src/main/kotlin/com/kotlin/athena/StartQueryExample.kt

```diff
@@ -128,14 +121,13 @@ suspend fun waitForQueryToComplete(athenaClient: AthenaClient, queryExecutionIdV
 suspend fun processResultRows(athenaClient: AthenaClient, queryExecutionIdVal: String?) {
     try {
 
-        // Max Results can be set but if it's not set, it will choose the maximum page size.
+        // Max Results can be set but if its not set, it will choose the maximum page size.

```

Comment:
it is or it's. You were right the first time.

Reply:
fixed